move-vm-backend/tests/move_vm.rs

```diff
@@ -62,6 +64,18 @@ fn estimate_gas_for_published_bytecode(bytecode: &[u8]) -> u64 {
     num_integer::div_ceil(raw_gas_cost, 1000)
 }
 
+fn store_preloaded_with_stdlib() -> StorageMock {
+    let genesis_cfg = VmGenesisConfig::default();
+    let store = StorageMock::new();
+
+    // Publish the stdlib.
+    assert!(
+        genesis_cfg.apply(store.clone()).is_ok(),
```

Comment:
Meh, TBH I was initially confused that we're cloning the storage and applying changes to the clone but returning the original one.
I needed to check how the StorageMock is done to see we're cloning pointer inside the mock...

Reply:
You don't need to do anything with that - it was just my feeling when I read the code :smile:

Reply:
haha no worries, my emoji answer was a way to sympathize with your feeling :smile: